src/routes/transactions/entities/schemas/propose-transaction.dto.schema.ts

```diff
@@ -0,0 +1,35 @@
+import { JSONSchemaType } from 'ajv';
+import { ProposeTransactionDto } from '../propose-transaction.dto.entity';
+
+export const proposeTransactionDtoSchema: JSONSchemaType<ProposeTransactionDto> =
+  {
+    $id: 'https://safe-client.safe.global/schemas/transactions/propose-transaction.dto.json',
+    type: 'object',
+    properties: {
+      to: { type: 'string' },
+      value: { type: 'string' },
+      data: { type: 'string', nullable: true },
+      nonce: { type: 'string' },
+      operation: { type: 'number', enum: [0, 1] },
+      safeTxGas: { type: 'string' },
+      baseGas: { type: 'string' },
+      gasPrice: { type: 'string' },
+      gasToken: { type: 'string' },
+      refundReceiver: { type: 'string', nullable: true },
+      safeTxHash: { type: 'string' },
+      sender: { type: 'string' },
+      signature: { type: 'string', nullable: true },
+      origin: { type: 'string', nullable: true },
+    },
+    required: [
+      'to',
+      'value',
+      'nonce',
+      'operation',
+      'safeTxGas',
+      'baseGas',
+      'gasPrice',
```

Comment:
Missing `gasToken` from the required parameters.

Reply:
Oops! Fixed in [dd0da0d](https://github.com/safe-global/safe-client-gateway-nest/pull/292/commits/dd0da0dd00d95a9fe17d7c2f5139e14d1c3565e3)